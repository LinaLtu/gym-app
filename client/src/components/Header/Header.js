import React from 'react';
import { useHistory } from 'react-router-dom';
import './Header.scss';

const CSS_NAME = 'header'

const Header = () => {

    const history = useHistory();

    const handleOnClick = () => {
        history.push("/");
    }

    return (
        <div className={CSS_NAME} onClick={handleOnClick}>
            Gymondo
            <div className={`${CSS_NAME}_logo`}></div>
        </div>
    )
}

export default Header;