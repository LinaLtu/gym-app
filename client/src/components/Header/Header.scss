@import '../shared/common.scss';

.header {
    width: 100%;
    height: 70px;
    background-color: $brand_color;

    &_logo {
        height: 100px;
        width: 100px;
        background-image: url('./logo.svg');
        background-repeat: no-repeat;
    }
}