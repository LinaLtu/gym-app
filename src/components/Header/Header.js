import React from 'react';
import './Header.scss';

const CSS_NAME = 'header'

const Header = () => {
    return (
        <div className={CSS_NAME}>
            Gymondo
            <div className={`${CSS_NAME}_logo`}></div>
        </div>
    )
}

export default Header;