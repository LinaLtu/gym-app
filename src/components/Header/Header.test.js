import React from 'react';
import renderer from 'react-test-renderer';

import Header from './';


describe('Header Component', () => {
    it('renders according to design', () => {
        const component = renderer.create(
            <Header />
        );
        const tree = component.toJSON();

        expect(tree).toMatchSnapshot();
    });
});