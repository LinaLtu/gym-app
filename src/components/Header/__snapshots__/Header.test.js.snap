// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component renders according to design 1`] = `
<div
  className="header"
>
  Gymondo
  <div
    className="header_logo"
  />
</div>
`;
